syntax/std_strconv.go

```diff
@@ -0,0 +1,22 @@
+package syntax
+
+import (
+	"github.com/arr-ai/arrai/rel"
+)
+
+func stdStrconv() rel.Attr {
+	return rel.NewAttr("strconv", rel.NewTuple(
+		//TODO: eval needs to be changed to only evaluate simple expression
+		// e.g. no functions, no math operations etc only simple values
+		rel.NewNativeFunctionAttr("eval", unsafeEval),
+		rel.NewNativeFunctionAttr("unsafe_eval", unsafeEval),
```

Comment:
The signature for `unsafe_eval` should be `\scope \expr ...`, where `scope` is a tuple of bound names to values to be used as the scope for `expr`.